tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/bigquery_tests_base.py

```diff
@@ -331,6 +331,22 @@ def default_test(self, tables_and_test_values):
                     fields = [table_info.get('fields', [])[0]]
                     self.assertRowIDsMatch(fq_sandbox_name, fields, values)
 
+        def copy_vocab_tables(self, vocabulary_id):
+            """
+            A function for copying the vocab tables to the test dataset_id
+            :param vocabulary_id:
+            :return:
+            """
+            # Copy vocab tables over to the test dataset
+            vocabulary_dataset = self.client.get_dataset(vocabulary_id)
+            for src_table in self.client.list_tables(vocabulary_dataset):
+                schema = self.client.get_table_schema(src_table.table_id)
+                destination = f'{self.project_id}.{self.dataset_id}.{src_table.table_id}'
+                dst_table = self.client.create_table(bigquery.Table(
+                    destination, schema=schema),
+                                                     exists_ok=True)
```

Comment:
I see a BigQuery error due to a schema mismatch `Provided Schema does not match Table test_project:test_rdr.vocabulary. Field vocabulary_reference has changed mode from REQUIRED to NULLABLE`. Not sure if this is something we need to update in our schemas?

Reply:
Interesting.  This changed from [5.3](https://ohdsi.github.io/CommonDataModel/cdm53.html#VOCABULARY) to [5.4](https://ohdsi.github.io/CommonDataModel/cdm54.html#VOCABULARY).  Our schema is still on the weird 5.3.1.  If 5.4 is the new 5.3.1, then we may need a ticket to update this.  Is this covered in the list of issues Jason found?  I guess we can ask him.

Reply:
@jp3477 Do you know if this is one of the fields that was modified? Not sure if you looked at vocabulary tables for the schema differences.

Reply:
This didn't show up on my list because I did a comparison with version v5.3. Should I run another comparison with v5.4 and see?

Reply:
Ah, I see. Thanks Jason! I think we should be okay for now. Lauren, this was my fault, the VOCABULARY_DATASET env var was pointing to an old vocabulary dataset. When I updated it to the latest one, the error disappeared.